salt/modules/pdbedit.py

```diff
@@ -39,14 +40,21 @@ def __virtual__():
     '''
     Provides pdbedit if available
     '''
-    if salt.utils.path.which('pdbedit'):
-        return __virtualname__
-    return (
-        False,
-        '{0} module can only be loaded when pdbedit is available'.format(
-            __virtualname__
-        )
-    )
+    # NOTE: check for pdbedit command
+    if not salt.utils.path.which('pdbedit'):
+        return (False, 'pdbedit command is not available')
+
+    # NOTE: check version is >= 4.8.x
+    ver = __salt__['cmd.run']('pdbedit -V')
```

Comment:
@garethgreenaway maybe you can correct my thinking here, but i believe using `__salt__` in `__virtual__` cannot always be reliable/guaranteed because `__virtual__` is called before you can ensure all other modules have been loaded?

Reply:
As per slack:
```
20:12 ch3ll okay gareth just got back to me, he agrees we shouldn't be using `_salt_` he said he thinks importing salt.modules.cmdmod is the best/safest approach
```